tests/MyGame/Example/Color.cs

```diff
@@ -8,7 +8,10 @@ namespace MyGame.Example
 public enum Color : byte
 {
  Red = 1,
+  /// \brief color Green
```

Comment:
appears misaligned, but that is the fault of the enum itself.. might as well fix?

Reply:
Probably, the comments have the right indent (2 spaces), while the enum values are misaligned (1 space).
Fix indent of values or adjust indent of comments?

Reply:
I'd say fix the values, which are clearly incorrect.

Reply:
Oh, I missed your remark about indent of enumeration values.
Fixed.